nixos/modules/services/continuous-integration/github-runner.nix

```diff
@@ -213,17 +244,28 @@ in
               if [[ -e "$STATE_DIRECTORY/${newConfigTokenFilename}" ]]; then
                 echo "Configuring GitHub Actions Runner"
 
-                token=$(< "$STATE_DIRECTORY"/${newConfigTokenFilename})
-                RUNNER_ROOT="$STATE_DIRECTORY" ${cfg.package}/bin/config.sh \
```

Comment:
Was the mismatch between this command being run and the one set in `environment` were causing issues? (Seems odd that it would ever work in the first place if that's the case, but hey, edge cases!)

As the original author of this module, why did you mismatch these two in the first place? Was it a typo, or was there some underlying reason?

Reply:
This was an error from the start. I cannot really recall what was my reasoning then but I probably tried to keep as much data as possible in the runtime directory (which Systemd removes on service stop). It didn't matter until introducing ephemeral mode as the runner didn't use the `RUNNER_ROOT` path to address files during runtime.